File: Languages/Lua/九九乘法表/Table.lua
-- [Author: Paddy Huang]
-- [Date: 2019-10-07 06:24:17]
-- [Description: 九九乘法表]

for i = 1, 9 do
	local res = ""
	for j = 1, i do
		res = res .. string.format("%dX%d=%d ", i, j, i*j)
	end
	print(res)
end

File: Languages/C++/九九乘法表/Table.cpp
/*------Description------
Author: Paddy Huang
Date: 2019-10-06 22:03:57
Description: 九九乘法表
*/
#include <iostream>

int main(){
	for(int i = 1; i < 10; i++){
		for(int j = 1; j <= i; j++)
			std::cout << i << "X" << j << "=" << i*j << " ";
		std::cout << std::endl;
	}
	return 0;
}
